posts/v2-metrics.md

```diff
@@ -0,0 +1,54 @@
+
+One caveat is that V2 probably encourages people to sign up who aren't yet ready for Exercism as it stands. V2 isn't set up for total beginners, yet the landing page for V2 is much more welcoming for total beginners, so we probably get a lot of users who aren't yet ready for Exercism. One way to account for this would be to consider not only the number of visitors signing up, but rather how many visitors signed up _and_ managed to submit a solution. For this we see:
+
+V1: 26,194 (7% new visitors sign up and submit)
+V2: 52,533 (13.9% new visitors sign up and submit)
+
+So from this we can say that the branding has accounted for approximately a doubling of our true "conversion rate", which makes the rebranding effort definitely worthwhile. It's also worth noting that as we move forward to being beginner friendly (V3.x) this caveat will go away altogether.
+
+### Solutions getting feedback (Mentoring)
+
+The second thing we wanted to improve was the amount of mentoring per solution. In V1 it was a bit of a lottery - you might get comments or you might not, purely dependent on whether someone noticed your submission. A big part of the formal mentoring of V2 was about making this fairer, consistent and reliable.
+
+The metric I've used here is percentage of solutions that receive a comment.
+
+V1: 3.5% of all solutions
+V2: 64% of solutions where mentoring has been requested (26% of all solutions)
+
+This is the phenominal statistic in my eyes, and the big indicator that V2 took us in the right direction. You're now 18x more likely to recieve feedback on your solution in V2 than in V1. There's a lot we got wrong with mentoring in V2 (see the next section!) but I feel hugely proud and grateful to our mentors for the fact that Exercism offers a place where if you want mentoring, you'll more likely than not recieve it.
+
+### Stickiness
+
+Finally, we wanted to improve "stickiness" - the amount of usage of the site. If people are enjoying Exercism and finding it useful, they'll do more with it. There are two sides to this, the amount of times someone comes back to the site, and the amount of solutions someone uploads.
+
+V1: 4.79 sessions / person
+V2: 5.42 sessions / person
+
+V1: 8.0 solutions / person
+V2: 8.2 solutions / person
+
+Both held pretty much steady. This is one area where I hoped we'd see a big increase in V2 and we didn't. The wait times for mentoring, and the blocking mechanisms that stop progression without mentoring both made V2 a very slow experience. For someone wanting to get stuck into a language, with the huge mentor queues we often saw, it became difficult to learn at a reasonable rate. The experience here will be very track-dependent. For tracks with wait times longer than a day, the dropout rate is noticeable. For those with longer wait times (e.g. Rust, Python), the dropout rate is substantial.
```

Comment:
@iHiD "get stuck into" is a very British expression. That isn't bad, but I would suspect that most non-Brits would not actually know what it means.

Reply:
I am not British, had never heard the expression before, I thought it was simply the way Jeremy expressed becoming attached or getting entrenched with a language.  Learn something new everyday.  I attributed the expression more personally to him, than to some nation state.

Reply:
I've changed it - thanks. Had no idea it wasn't a global phrase :)